spec/helpers/application_helper_spec.rb

```diff
@@ -47,4 +47,28 @@
       expect(helper.table_of_contents_separator(input)).to match(/data-toggle='collapse'/)
     end
   end
+
+  describe '#manuscript_link' do
+    let(:title) { 'Baldwin of Ford OCist, De sacramento altaris' }
+    let(:show_page) { '/test-flag-exhibit-slug/catalog/bg021sq9590' }
+    let(:document) do
+      SolrDocument.new(
+        title_full_display: "p. 3:#{title}",
+        manuscript_number_tesim: ['MS 198'],
+        format_main_ssim: ['Page details']
+      )
+    end
+    let(:input) { { value: ['bg021sq9590'], document: document } }
+
+    before do
+      helper.extend(Module.new do
+        def current_exhibit; end
```

Comment:
Is this and https://github.com/sul-dlss/exhibits/pull/848/files#diff-31c50b16a4c9f23be71c49d30afcf7edR70 needed?

Reply:
@jkeck As far as I can tell, yes. I'm working off the example of: https://github.com/sul-dlss/exhibits/blob/master/spec/helpers/blacklight/maps_helper_override_spec.rb

Reply:
alright, one more go @jkeck. I was able to remove the `allow(helper)` and after adding `FactoryBot` to `helper.extend`.